src/plays/drumkit/styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Arvo:wght@100;200;300;400;500;600;700&display=swap');

```

Comment:
also, do not override check the docs & add a unique name 

https://docs.reactplay.io/how-to-guides/how-to-style-play/

Reply:
@priyankarpal okay.